dev/tests/integration/testsuite/Magento/Framework/Translate/InlineTest.php

```diff
@@ -119,7 +119,7 @@ public function processResponseBodyDataProvider()
         $package = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->get(
             \Magento\Framework\View\DesignInterface::class
         )->getDesignTheme()->getPackageCode();
-        $expectedText = str_replace('{{design_package}}', $package, $expectedText);
+        $expectedText = str_replace('{{design_package}}', is_string($package) ? $package : '', $expectedText);
```

Comment:
Could you please add more details why `$package` could be `null` ?

Reply:
Hello, @xmav, It can be set or not. The `\Magento\Theme\Model\Theme` extends DataObject, so potentially we can have here `null`.

Thank you.